act/workers/scio2.py

```diff
@@ -114,7 +115,7 @@ def add_indicators_to_act(actapi: act.api.Act,
             actapi.fact("represents")
             .source("hash", sha256)
             .destination("content", sha256),
-            output_format
+            output_format=output_format
```

Comment:
Not sure why this is needed. It works without this when testing manually, but the tests fail without this change.